api/apps/articles/model.coffee

```diff
@@ -255,3 +255,4 @@ addSlug = (article, input, author) ->
     _id: undefined
     slug: _.last article.slugs
     slugs: undefined
+    contributing_authors: article.contributing_authors or []
```

Comment:
I think ideally the database should try to reflect the json responses as much as possible so update in the db preferred. Use `db.articles.update({ contributing_authors: null }, { contributing_authors: [] }, { multi: true })` but make sure to try it on local/staging first. You can do this with robomongo or `heroku run coffee --app=positron-production` (the former will be easier).